### DISCUSSION TITLE
Swiper slider blank space on the right after last slide

### DISCUSSION THREAD

**Discussion by Drinosi** [nolimits4web/swiper#5540] [Q&A]
I have used swiper slide library to create multiple slides on my page. 4 total. The first 2 work fine, the last 2 however have this glitch where If I swipe enough on desktop the slides dont appear on the right side. This only happens on these occasions.

    If the mouse is being used to swipe.(not the prev and next buttons).
    If I have 5 slides visible in >1024px screens.(if I change it to 4 slides then its fine.)
    Only on desktop.

The page is https://www.hometownworkforce.com/ Please make sure to include www. or it will link to another page. You can use inspect element to check the code, the code is in a script tag at the very end of the main index.

**Comment by Arhell** [comment]
You have 2 sliders with the same classes, I think this is the problem
![Screenshot_1](https://user-images.githubusercontent.com/26163841/157747900-8179ff9e-0395-43fa-9a22-453cbd998dd5.png)
.

**Reply by Drinosi** [reply]
those arent the sliders, thanks but I figured what the problem was, I had 4 sliders with two classes, one called swiper and another swiper1/2/3/4, basically removing the "swiper" class for the other slides other than the first ones and only keeping the swiper/2/3/4 as it was called in the JS code fixed it.
